e2e/nomostest/config-sync.go

```diff
@@ -524,7 +524,7 @@ func ValidateMultiRepoDeployments(nt *NT) error {
 		nt.T.Logf("took %v to wait for %s %s", took, "ValidatingWebhookConfiguration", "admission-webhook.configsync.gke.io")
 	}
 
-	return validateMultiRepoPods(nt)
```

Comment:
We need to validate the Pod status because it is possible that the Deployment is validated, but the Pod is stuck in a crashloopbackoff state.

Reply:
Same comment as https://github.com/GoogleContainerTools/kpt-config-sync/pull/111#discussion_r974657873.